apps/heureka/src/components/filters/Filters.jsx

```diff
@@ -17,7 +17,7 @@ const filtersStyles = `
   my-px
 `
 
-const Filters = ({ queryKey, entityName }) => {
+const Filters = ({ queryKey, entityName, labels, filterLabelValues }) => {
```

Comment:
I would propose the same with the callbacks here and make this component entityName agnostic

Reply:
Same valuable optimization, and I implement it.